HW04/anastasia_evsihneyeva/lib/mentor.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Mentor
+  def initialize(data)
+    @name = data[:name]
+    @surname = data[:surname]
+    @students = []
+  end
+
+  def add_homework(data)
+    Notification.new(log: "-New homework was added! Name #{data[:title]}")
+    name = "#{@name} #{@surname}"
+    Homework.new(data, name)
+  end
+
+  def subscribe_to!(student)
+    Notification.new(log: "-Mentor #{@name} #{@surname} subscribed to #{student.name}")
+    @students << student
+  end
+
+  def notifications
+    Notification.new
+  end
+
+  def mark_as_read!
+    Notification.new(log: '-Mark_as_read!')
```

Comment:
If you are already doing such functionality, then it is expected that you will have a functionality that will store unread notifications, and then clear them after reading